automation/release.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -xe
+
+# This script tags current version and create a new release at github and pushes
+# to quay.io
+# To run it just do proper docker login and pass the git hub user password as
+# GIT_USER and GIT_PASSWORD env variables.
```

Comment:
You mention GIT_USER and GIT_PASSWORD, but the function uses GITHUB_USER and GITHUB_TOKEN